cloudinit/util.py

```diff
@@ -1166,7 +1166,7 @@ def read_cc_from_cmdline(cmdline=None):
     if cmdline is None:
         cmdline = get_cmdline()
 
-    tag_begin = "cc:"
+    tag_begin = " cc:"
```

Comment:
Since it is valid for the line to begin with `cc:`, this won't work in that case.

I'm tempted to replace this entire section with regex parsing.